docs/dev-proc/source-code-structure.md

```diff
@@ -67,30 +67,27 @@ exception of rebasing the `dasharo` branch on new coreboot versions.
 
     ```bash
     git checkout dasharo
-    git checkout -b common-base-4.21
-    git push -u origin common-base-4.21
+    git checkout -b dasharo-4.21
+    git push -u origin dasharo-4.21
     ```
 
 1. Rebase the current `dasharo` branch on the latest tag, for example:
 
     ```bash
-    git checkout dasharo
+    git checkout -b dasharo-4.22
     git rebase 4.22
-    ```
-
-    If there are problems with rebase, you may create a different branch, e.g.
-    `common-base-4.22` and work until all issues are resolved, then rename the
-    branch back to dasharo
-
-    ```bash
-    git checkout -b common-base-4.22
     # resolve issues
```

Comment:
Just for sake of completeness, I'd add some kind of testing here, at least build tests.

Reply:
Added a step in e06d50d190857ceb3a435bef83fb91cf948899af